test/scripts/jenkins_win_selenium.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/bash
+
+source "$(dirname $0)/../../src/dev/ci_setup/setup.sh"
+ 
+# Install Dir
+installDir=$(pwd)
+mkdir -p "$installDir" 
+
+# Get Version
+pkg_version=$(cat package.json | grep "\"version\"" | cut -d ':' -f 2 | tr -d ",\"\ +")
```

Comment:
We can rely on the jq package being available, which makes this a lot cleaner

```suggestion
pkg_version="$(jq -r .version package.json)"
```